classes/task/email_certificate_task.php

```diff
@@ -248,8 +311,18 @@ public function execute() {
                 }
 
                 // Set the field so that it is emailed.
-                $DB->set_field('customcert_issues', 'emailed', 1, ['id' => $user->issueid]);
+                $issueids[] = $user->issueid;
+            }
+            if (!empty($issueids)) {
+                $DB->set_field_select('customcert_issues', 'emailed', 1, 'id IN (' . implode(',', $issueids) . ')');
```

Comment:
You need to use `$DB->get_in_or_equal` here. See example at https://github.com/mdjnelson/moodle-mod_customcert/blob/MOODLE_402_STABLE/classes/certificate.php#L467.

Reply:
Done